DataConnectors/Digital shadows/DigitalShadowsConnectorAzureFunction/DS_api.py

```diff
@@ -4,32 +4,51 @@
 
 
 class api:
-    
-    url = "https://api.searchlight.app/v1/"                 #main url for api
 
-    """ constructer initializes the DS creds """
-    def __init__(self, id, key, secret):
+    def __init__(self, id, key, secret, uri):
+        """ 
+            constructer initializes the DS creds and creates passkey
+        """
+
+        self.url = "https://" + str(uri) + "/"
```

Comment:
What is user specifies the server name with scheme (https). Better use https://docs.python.org/3/library/urllib.parse.html